README.md

```diff
@@ -19,8 +19,10 @@ Yet it is very easy to use. You need learn only a few things.
 1. You have to always pass variables into the query via *placeholder*
 2. Each placeholder has to be marked with its data type. At the moment there are 6 types:
  * `?s` **("string")**              - strings (also `DATE`, `FLOAT` and `DECIMAL`)
+ * `?b` **("boolean")**             - booleans (can accept true, false, 1, 0, "1", or "0")
  * `?i` **("integer")**             - integers
- * `?n` **("name")**                - identifiers (table and field names) 
+ * `?n` **("name")**                - identifiers (table and field names)
```

Comment:
- [ ] There should be an "e.g." at the beginning of these brackets; identifiers are not limited to table and field names (though those are the most typical cases) but also include database names, function and sproc names, and probably a whole bunch of other stuff I don't know about.